Index: postr/Makefile.in
===================================================================
--- postr.orig/Makefile.in
+++ postr/Makefile.in
@@ -40,8 +40,7 @@ DIST_COMMON = README $(am__configure_dep
 	$(dist_appicon22_DATA) $(dist_appicon24_DATA) \
 	$(dist_appicon32_DATA) $(dist_appiconsc_DATA) \
 	$(dist_noinst_DATA) $(dist_noinst_SCRIPTS) $(postr_PYTHON) \
-	$(postrnautilus_PYTHON) $(srcdir)/Makefile.am \
-	$(srcdir)/Makefile.in $(top_srcdir)/configure \
+	$(srcdir)/Makefile.am \ $(srcdir)/Makefile.in $(top_srcdir)/configure \
 	$(top_srcdir)/data/postr.desktop.in.in ABOUT-NLS AUTHORS \
 	COPYING ChangeLog INSTALL NEWS TODO config.guess config.rpath \
 	config.sub install-sh missing py-compile
@@ -88,7 +87,7 @@ am__uninstall_files_from_dir = { \
          $(am__cd) "$$dir" && rm -f $$files; }; \
   }
 am__installdirs = "$(DESTDIR)$(bindir)" "$(DESTDIR)$(postrdir)" \
-	"$(DESTDIR)$(postrnautilusdir)" "$(DESTDIR)$(desktopdir)" \
+	"$(DESTDIR)$(desktopdir)" \
 	"$(DESTDIR)$(appicon16dir)" "$(DESTDIR)$(appicon22dir)" \
 	"$(DESTDIR)$(appicon24dir)" "$(DESTDIR)$(appicon32dir)" \
 	"$(DESTDIR)$(appiconscdir)" "$(DESTDIR)$(uidir)"
@@ -279,7 +278,6 @@ localedir = @localedir@
 localstatedir = @localstatedir@
 mandir = @mandir@
 mkdir_p = @mkdir_p@
-nautilusextensiondir = @nautilusextensiondir@
 oldincludedir = @oldincludedir@
 pdfdir = @pdfdir@
 pkgpyexecdir = @pkgpyexecdir@
@@ -330,10 +328,6 @@ postr_PYTHON = \
 	src/version.py
 
 
-# Nautilus extension
-postrnautilusdir = $(nautilusextensiondir)/
-postrnautilus_PYTHON = nautilus/postrExtension.py
-
 # icons
 iconthemedir = $(datadir)/icons/hicolor
 appicon16dir = $(iconthemedir)/16x16/apps
@@ -506,41 +500,7 @@ uninstall-postrPYTHON:
 	  $(am__uninstall_files_from_dir) || st=$$?; \
 	done; \
 	exit $$st
-install-postrnautilusPYTHON: $(postrnautilus_PYTHON)
-	@$(NORMAL_INSTALL)
-	test -z "$(postrnautilusdir)" || $(MKDIR_P) "$(DESTDIR)$(postrnautilusdir)"
-	@list='$(postrnautilus_PYTHON)'; dlist=; list2=; test -n "$(postrnautilusdir)" || list=; \
-	for p in $$list; do \
-	  if test -f "$$p"; then b=; else b="$(srcdir)/"; fi; \
-	  if test -f $$b$$p; then \
-	    $(am__strip_dir) \
-	    dlist="$$dlist $$f"; \
-	    list2="$$list2 $$b$$p"; \
-	  else :; fi; \
-	done; \
-	for file in $$list2; do echo $$file; done | $(am__base_list) | \
-	while read files; do \
-	  echo " $(INSTALL_DATA) $$files '$(DESTDIR)$(postrnautilusdir)'"; \
-	  $(INSTALL_DATA) $$files "$(DESTDIR)$(postrnautilusdir)" || exit $$?; \
-	done || exit $$?; \
-	if test -n "$$dlist"; then \
-	  $(am__py_compile) --destdir "$(DESTDIR)" \
-	                    --basedir "$(postrnautilusdir)" $$dlist; \
-	else :; fi
 
-uninstall-postrnautilusPYTHON:
-	@$(NORMAL_UNINSTALL)
-	@list='$(postrnautilus_PYTHON)'; test -n "$(postrnautilusdir)" || list=; \
-	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
-	test -n "$$files" || exit 0; \
-	dir='$(DESTDIR)$(postrnautilusdir)'; \
-	filesc=`echo "$$files" | sed 's|$$|c|'`; \
-	fileso=`echo "$$files" | sed 's|$$|o|'`; \
-	st=0; \
-	for files in "$$files" "$$filesc" "$$fileso"; do \
-	  $(am__uninstall_files_from_dir) || st=$$?; \
-	done; \
-	exit $$st
 install-desktopDATA: $(desktop_DATA)
 	@$(NORMAL_INSTALL)
 	test -z "$(desktopdir)" || $(MKDIR_P) "$(DESTDIR)$(desktopdir)"
@@ -998,7 +958,7 @@ check: check-recursive
 all-am: Makefile $(SCRIPTS) $(DATA)
 installdirs: installdirs-recursive
 installdirs-am:
-	for dir in "$(DESTDIR)$(bindir)" "$(DESTDIR)$(postrdir)" "$(DESTDIR)$(postrnautilusdir)" "$(DESTDIR)$(desktopdir)" "$(DESTDIR)$(appicon16dir)" "$(DESTDIR)$(appicon22dir)" "$(DESTDIR)$(appicon24dir)" "$(DESTDIR)$(appicon32dir)" "$(DESTDIR)$(appiconscdir)" "$(DESTDIR)$(uidir)"; do \
+	for dir in "$(DESTDIR)$(bindir)" "$(DESTDIR)$(postrdir)" "$(DESTDIR)$(desktopdir)" "$(DESTDIR)$(appicon16dir)" "$(DESTDIR)$(appicon22dir)" "$(DESTDIR)$(appicon24dir)" "$(DESTDIR)$(appicon32dir)" "$(DESTDIR)$(appiconscdir)" "$(DESTDIR)$(uidir)"; do \
 	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
 	done
 install: install-recursive
@@ -1058,7 +1018,7 @@ info-am:
 install-data-am: install-desktopDATA install-dist_appicon16DATA \
 	install-dist_appicon22DATA install-dist_appicon24DATA \
 	install-dist_appicon32DATA install-dist_appiconscDATA \
-	install-postrPYTHON install-postrnautilusPYTHON install-uiDATA
+	install-postrPYTHON install-uiDATA
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-recursive
@@ -1110,7 +1070,7 @@ uninstall-am: uninstall-binSCRIPTS unins
 	uninstall-dist_appicon16DATA uninstall-dist_appicon22DATA \
 	uninstall-dist_appicon24DATA uninstall-dist_appicon32DATA \
 	uninstall-dist_appiconscDATA uninstall-postrPYTHON \
-	uninstall-postrnautilusPYTHON uninstall-uiDATA
+	uninstall-uiDATA
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) uninstall-hook
 .MAKE: $(RECURSIVE_CLEAN_TARGETS) $(RECURSIVE_TARGETS) ctags-recursive \
@@ -1131,7 +1091,7 @@ uninstall-am: uninstall-binSCRIPTS unins
 	install-dvi install-dvi-am install-exec install-exec-am \
 	install-html install-html-am install-info install-info-am \
 	install-man install-pdf install-pdf-am install-postrPYTHON \
-	install-postrnautilusPYTHON install-ps install-ps-am \
+	install-ps install-ps-am \
 	install-strip install-uiDATA installcheck installcheck-am \
 	installdirs installdirs-am maintainer-clean \
 	maintainer-clean-generic maintainer-clean-local mostlyclean \
@@ -1141,7 +1101,7 @@ uninstall-am: uninstall-binSCRIPTS unins
 	uninstall-dist_appicon22DATA uninstall-dist_appicon24DATA \
 	uninstall-dist_appicon32DATA uninstall-dist_appiconscDATA \
 	uninstall-hook uninstall-postrPYTHON \
-	uninstall-postrnautilusPYTHON uninstall-uiDATA
+	uninstall-uiDATA
 
 
 # desktop file & translation
